.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,42 @@
+<!-- Please use this template when creating an issue. 
+- Please check the boxes after you've created your issue.
+- Please use the latest version of Accessibility Checker.-->
+
+* [ ] I've read and understood the [contribution guidelines](https://github.com/equalizedigital/accessibility-checker/blob/develop/.github/CONTRIBUTING.md).
+* [ ] I've searched for any related issues and avoided creating a duplicate issue.
+
+### Please give us a description of what happened.
+
+
+
+
+### Please describe what you expected to happen and why.
+
+
+
+
+### How can we reproduce this behavior?
+1.
+2.
+3.
+
+### Technical info
+* WordPress version:
+* Accessibility Checker version:
+<!-- You can check these boxes once you've created the issue. -->
+* If relevant, which editor is affected (or editors): 
+- [ ] Classic Editor
+- [ ] Gutenberg
+- [ ] Classic Editor plugin
+
+<!-- You can check these boxes once you've created the issue. -->
+* Which browser is affected (or browsers): 
+- [ ] Internet Explorer
+- [ ] Edge
+- [ ] Chrome
+- [ ] Firefox
+- [ ] Safari
+
+* Relevant plugins in case of a bug:
+<!-- Please make sure you can reproduce this bug with a default theme such as Twenty Seventeen. Sometimes issues may occur due to theme conflicts. -->
```

Comment:
It's fine to address later. My thought is depending upon the bug (like examples of false positives on the ruleset) - switching themes may very well make it go away because the issue is in the theme.